_activities/gsoc.md

```diff
@@ -24,47 +24,120 @@ To answer these questions, particle physicists build software to simulate and an
 
-For further information, feel free to contact the HSF GSoC admins or join our new chat room
+For further information, feel free to contact the HSF GSoC admins or join our open chat channel below. Note that the channel is intended for exchanging with the admins, the other candidates and some of the CERN-HSF students from previous year, and we do not maintain a mailing list for candidates. Please avoid posting extended information about yourself in the chat channel and reserve this for the direct exchange with the mentors.
 
 [hsf-gsoc-admin@googlegroups.com](mailto:hsf-gsoc-admin@googlegroups.com)
 
 [![Gitter](https://badges.gitter.im/HSF/HSF-GSoC.svg)](https://gitter.im/HSF/HSF-GSoC?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge)
 
 ### For Projects and Mentors
 
-Instructions for participating projects and mentors can be found [here](/gsoc/guideline.html).
-
-We have a Google group where we handle all future announcements related to GSoC for our HSF Umbrella Organization. Please write us a mail to [hsf-gsoc-admin@googlegroups.com](mailto:hsf-gsoc-admin@googlegroups.com) in case you are interested to add your preferred mail to this list. Note that all 2020 HSF GSoC mentors will have to join this group.
-
-For new HEP-related groups wishing to join HSF GSoC umbrella rather than being independent organizations: there is no formal procedure, you will just need to add a description and logo for your organization. We need however one contact person for each new group, so please write us an e-mail.
-
-## Projects in 2020
-
-{% assign current_year = "2020" %}
+Instructions for participating projects and mentors can be found [here](/gsoc/guideline.html). The main differences compared to last year are that proposals have to be tuned for 175-hour project length, and the target objectives must be well-defined deliverables.
+
+We have a mentor-only Google group where we handle all announcements related to GSoC for our HSF Umbrella Organization. Please write us a mail to [hsf-gsoc-admin@googlegroups.com](mailto:hsf-gsoc-admin@googlegroups.com) in case you are interested to add your preferred mail to this list. Note that all participating mentors in 2021 HSF GSoC will have to be added to this group. Please let us know if for some reason your want your mail removed from this list.
```

Comment:
"Please write us a mail" => "Please write us mail" ? (Unnecessary article – Since mail is uncountable, it cannot be used with a. Dixit my corrector … ?!)

Reply:
"for some reason your want" => "for some reason you want"